### PR TITLE
chore/importbin

### PR DESCRIPTION
**Pull request by primeroz**
Replace import binary:// with importbin in our tests

### CODE REVIEW

**Review by mkmik — APPROVED** [review]

File: utils/importer.go [hunk 1]
```diff
@@ -98,6 +98,10 @@ func (importer *universalImporter) Import(importedFrom, importedPath string) (js
 	var tried []string
 	for _, u := range candidateURLs {
 		foundAt := u.String()
+		// Avoid collision bug when importing same chart in the same jsonnet file using `import binary://` and `importbin`
+		if binary {
+			foundAt = "binary-" + u.String()
```
Comment (mkmik) [inline utils/importer.go:103]: what happens if the imported file name starts with `binary-` ?

Comment (primeroz) [reply utils/importer.go:103]: it breaks, let me see to do it more cleanly by using the original binary:// prefix

Comment (primeroz) [reply utils/importer.go]: i changed it to a suffix like `##binaryImport` to the filename. i think the risk of someone files being called `CHARTNAME.tgz##binaryImport` are close to 0 :)

**Review by mkmik — APPROVED** [review]
it's better; although one could have escaped `#` in the url; but it's very very unlikely to clash and we're going to deprecate the `binary:/` hack soon

**Pull request merged**